how-to/workspace-platform-starter/public/manifest.fin.json

```diff
@@ -270,22 +269,26 @@
 				{
 					"id": "favorite-list",
 					"type": "module",
-					"typeId": "favorite-local-storage"
+					"typeId": "favorite-local-storage",
+					"options": {}
```

Comment:
Why do we add empty options, shouldn't options be optional ?

Reply:
The endpoint options is mandatory. Updated to only be mandatory for type="fetch"and updated code that broke as a result. Updated the remaining manifest to not have a reference to the schema and updated the naming to reflect naming convention of *.manifest.fin.json